GSOC18/CNN_Module/utils/conv2d_utils.py

```diff
@@ -37,14 +37,13 @@ def get_variable_on_cpu(name,shape,initializer,weight_decay=None):
         #Applying the l2 regularization and multiplying with
         #the hyperparameter weight_decay: lambda
         reg_loss=tf.multiply(tf.nn.l2_loss(weight),weight_decay,
-                                name='lambd_hyparam')
```

Comment:
Please don't change this. This is how "lambda" is referred elsewhere in code, otherwise, the name lambda is a keyword in python.

Reply:
okay